### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by kofimeighan**
Finished off the majority of the introduction page, and added the Education and Activities Page. Minor changes in the css file that dealt with unnecessary lines.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -36,13 +36,13 @@ <h3><b>Who am I?</b></h3>
 
     <div class="row">
         <div class="column">
-            <img src="images/pink!.jpg" style="width:100%">
+            <img src="images/pink!.jpg" style="width:100%" >
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/index.html:39]: Nitpicking: remove these whitespace before ">"?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 
 
-  <body bgcolor ="AliceBlue">
+  <body  class ="serif" bgcolor ="AliceBlue">
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/index.html:12]: Nitpicking: try to keep the single-space-between-attributes styling consistent. That means remove the extra space between 'class'.

Comment (wangtong-g) [reply portfolio/src/main/webapp/index.html:12]: Quick question: what does "serif" stand for?

File: portfolio/src/main/webapp/education.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Education & Activities</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="AliceBlue"> 
+      <h3 align="center">Education & Activities</h3>
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:11]: Nit: try to keep the indentation consistent for the file. Either two or four spaces.

Comment (wangtong-g) [reply portfolio/src/main/webapp/education.html:11]: My bad, html style guide says we should indent by 2 spaces.

File: portfolio/src/main/webapp/education.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    <ul class="nav">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="pictures.html">Film Pictures</a></li>
+        <li><a href="mixes.html">Music Mixes</a></li>
+    </ul>
+
+
+    <h4><u>Education</u></h4>
+    <p><i>Pine Crest School</i> - Pine Crest School is a private prepatory
+     school in Fort Lauderdale, Florida. I attended from August 2015 to
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:29]: I think this section needs some formatting. Try moma "html style guide" internally.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/index.html:12]: Serif is a font family. I was applying it using a function within my style.css file.

File: portfolio/src/main/webapp/education.html (hunk 4 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/education.html:29]: Just tried searching moma, I don't have access to the actual style guide.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (wangtong-g) [reply portfolio/src/main/webapp/index.html]: I see. Thanks.

File: portfolio/src/main/webapp/education.html (hunk 4 above)
Comment (wangtong-g) [reply portfolio/src/main/webapp/education.html:29]: That is unfortunate... okay, I will try my best to read them and enforce..

**Pull request merged**